knowledge/science/anatomy/wisdom_teeth/qna.yml

```diff
@@ -0,0 +1,21 @@
+created_by: 
+domain: anatomy
+seed_examples:
+- answer: | The third molar, commonly called wisdom tooth, is the most posterior of the three molars in each quadrant of the human dentition.
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!